bridge/src/main/java/com/livefront/bridge/BridgeDelegate.java

```diff
@@ -144,6 +145,26 @@ private boolean isAppInForeground() {
         return mActivityCount > 0 || mIsConfigChange;
     }
 
+    /**
+     * If it's a fresh start or back to main activity, we can safely clear cache
+     */
+    private boolean isFreshStartOrBackToMain(@NonNull Activity activity, Bundle savedInstanceState) {
```

Comment:
We should add `@Nullable` to `Bundle` here.

Reply:
Question about the naming here...what do you mean exactly by `...OrBackToMain`? Aren't we really just trying to detect the "fresh start"?